docs/build/provision/_index.md

```diff
@@ -227,6 +227,13 @@ To provision your machine, create a <file>/etc/viam-provisioning.json</file> con
 
 This file configures some basic metadata, specifies a [fragment](/fleet/configure-a-fleet/) to use to configure the machine, and provides the WiFi network name and password to allow your machine to connect automatically on startup.
 
+{{% alert title="Support Notice" color="note" %}}
+
+You can add a fragment to your machine using Viam Agent.
+However, you cannot configure the agent itself inside of a fragment.
```

Comment:
Mostly good! But I think we just talked about adding a fragment to your machine via agent in the immediately preceeding example, so no need to call it out again, imo. Possibly just:

```suggestion
You cannot configure the Viam Agent itself using fragments.
```

Reply:
Okay I originally had the same thought so if you agree hopefully this is enough!